github_actions/run.py

```diff
@@ -55,6 +65,67 @@ def _handle_push_event(event: GithubEvent) -> None:
         raise EnvironmentError("Unable to retrieve From hash and To hash") from None
 
 
+def _write_output(name: str, value: Union[str, int]) -> None:
+    """
+    Writes an output to the GitHub Actions environment.
+
+    Args:
+        name (str): The name of the output variable.
+        value: The value to be assigned to the output variable.
+
+    Raises:
+        OSError: If there is an issue opening or writing to the output file.
+    """
+    output_file_path = os.environ.get("GITHUB_OUTPUT", "")
+    with open(file=output_file_path, mode="a", encoding="utf-8") as output_file:
+        print(f"{name}={value}", file=output_file)
```

Comment:
suggestion: We do not use print in our project. Please use the IO interfaces for writing.

```suggestion
        output_file.write(f"{name}={value}\n")
```